### PR TITLE
[DOCFIX] Update to correct short version

### PR DESCRIPTION
**Pull request by waltonryan**
@aaudiber This is a fix to display the correct version for the docs on alluxio.org.

### CODE REVIEW

**Comment by waltonryan** [comment]
@calvinjia

**Comment by calvinjia** [comment]
LGTM, thanks.

File: docs/_config.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ kramdown:
 ALLUXIO_RELEASED_VERSION: 1.2.0
 # This would be "master" for the master branch, 1.1.1 for the 1.1.1 release, and
 # 1.1.1-RC2 for the second 1.1.1 release candidate.
```
Comment (aaudiber) [inline docs/_config.yml]: we should update the comment too if we're not going to include the `-RC2` part any more

Comment (waltonryan) [inline docs/_config.yml]: Sounds good. I'll test the org site docs build and if builds correctly, will submit a separate PR if that works for you.

**Pull request merged**